### PR TITLE
master-nightly: add IPAAPI user/group during image creation

### PR DESCRIPTION
**Pull request by martbab**

### CODE REVIEW

File: Dockerfile.fedora-25-master-nightly [hunk 1]
```diff
@@ -53,6 +53,7 @@ RUN systemctl enable ipa-server-configure-first.service
 COPY exit-via-chroot.conf /usr/lib/systemd/system/systemd-poweroff.service.d/
 
 RUN groupadd -g 288 kdcproxy ; useradd -u 288 -g 288 -c 'IPA KDC Proxy User' -d '/var/lib/kdcproxy' -s '/sbin/nologin' kdcproxy
+RUN groupadd -r ipaapi ; useradd -g ipaapi -c 'IPA Management Framework User' -d '/var/lib' -s '/sbin/nologin' ipaapi
```
Comment (adelton) [inline Dockerfile.fedora-25-master-nightly]: Please specify uid / gid.

You have to do it or next image might pick different numbers and when you then bind-mount volume which has data from previous image, file ownership will be wrong.

Unlike of on-host installations where you install once and then upgrade software and keep the uid/gid if there already is one, with container there are no upgrades happening.

Comment (martbab) [inline Dockerfile.fedora-25-master-nightly]: @adelton thank you for reminding me that. I will fix it once I find out some free UIDs/GIDs for system users.

Comment (adelton) [inline Dockerfile.fedora-25-master-nightly]: Given that FPC was not keen on allocating uids for container use and they are also opposed to allocating them in general, per recommendation in https://pagure.io/packaging-committee/issue/474 we might need to either find someone to manage the list for us, or self-proclaim us to be the list maintainers.

**Comment by martbab** [comment]
Since https://github.com/freeipa/freeipa/pull/697/commits/eccc5ff2b80f70bf4658fce259a90b7fdd63409b has been pushed, we need to change our approach and instead re-number existing users/groups to a consistent UIDs/GIDs now. This PR is thus obsoleted and I will prepare a new one.

**Comment by adelton** [comment]
> instead re-number existing users/groups to a consistent UIDs/GIDs now.

Well, the question is, shouldn't we create those user/group records before doing the `dnf/yum install`, to avoid the need to re-number stuff in `/etc/passwd / /etc/group` and on the filesystem.

Ideally though, it'd be good to make sure that that unfortunate half-finished change to FreeIPA is finalized and corrected as soon as possible. Or the issue at least tracked.

**Pull request closed without merging**